### DISCUSSION TITLE
Sometimes the feed only shows activity related to me or my projects

### DISCUSSION THREAD

**Discussion by pawamoy** [community/community#130722] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

It's been multiple times now (maybe 10) that some days, ramdomly, my feed only shows activity related to me or my projects.

<table><tr><td>

![feedstars](https://github.com/community/community/assets/3999221/a83e74b9-df7b-4976-857f-454958fa833a)

</td><td>

![feedstars2](https://github.com/community/community/assets/3999221/e622bdfe-5089-452c-8896-99218778d9ae)

</td></tr></table>

Is GitHub trying to make me feel good about myself?

Usually it goes back to normal after a few hours, or the next day or something (not sure exactly).

**Comment by XpTo2k** [comment]
The same happens to me!

**Comment by joshistoast** [comment]
Same happens to me, I temporarily solve this by hiding starred activity in the filter, then once the feed is refreshed, re-enabling again. Though, this gets quickly undone by visiting a repo then going back to feed.
